src/shared/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+class Utils {
+
+    static formatDate(dateStr: string): string {
+        const date = new Date(dateStr);
+
+        return moment(date).format("YYYY MM DD");
```

Comment:
Creating a `Date` object is redundant since you can directly pass `dateStr` into `moment()`.

```suggestion
        return moment(dateStr).format("YYYY MM DD");
```